bitnami/thanos/templates/query-frontend/ingress.yaml

```
@@ -45,7 +45,7 @@ spec:
     {{- if or .Values.queryFrontend.ingress.secrets .Values.queryFrontend.ingress.tls }}
     - hosts:
         - {{ .Values.queryFrontend.ingress.hostname }}
-      secretName: {{ printf "%s-tls" .Values.queryFrontend.ingress.hostname }}
+      secretName: {{ include "common.names.fullname" . }}-query-frontend
```

You can still use "printf":
```suggestion
      secretName: {{ printf "%s-query-frontend" (include "common.names.fullname" .) }}
```